saw-core/src/SAWCore/SCTypeCheck.hs

```diff
@@ -294,7 +295,7 @@ prettyTCError e = runReader (helper e) ([], Nothing) where
   ishow :: Term -> PPErrM String
   ishow tm =
     -- return $ show tm
-    (\(ctx,_) -> indent "  " $ scPrettyTermInCtx PPS.defaultOpts ctx tm) <$> ask
+    (\(_ctx,_) -> indent "  " $ scPrettyTermInCtx PPS.defaultOpts []{-ctx-} tm) <$> ask
```

Comment:
Remove the `{-ctx-}` comment here?

(Similar comments apply to the various bits of commented-out stubs of code throughout this module, e.g., `{- withVar x (typedVal a_whnf) $ -}` and `{- withVar x1 a1 -}`.)